### DISCUSSION TITLE
Hourly service

### DISCUSSION THREAD

**Discussion by SheiylaDev** [AdguardTeam/AdGuardHome#5553] [Ideas]
Hello, it would be nice to be able to automatically activate the blocking of services that we want with a time range and day of the week !!!!

**Comment by fizsef** [comment]
Yes, a very necessary option. In the user rules, so that you can block on a schedule.

**Comment by reichelper** [comment]
That's a really good idea. I'm also interested in that feature.

**Comment by ainar-g** [comment]
See #951.  It's being worked on.
